### ISSUE TITLE
user test feedback

### ISSUE THREAD

**Issue by 79-6d** [GenomicsStandardsConsortium/gensc.github.io#48]
Hey,

These are the issues I found when testing this website: [https://genomicsstandardsconsortium.github.io/gensc.github.io/index.html](https://genomicsstandardsconsortium.github.io/gensc.github.io/index.html)

I didn't manage to test everything but these are the issue I found so far. Thanks a lot.

### [Home](https://genomicsstandardsconsortium.github.io/gensc.github.io/index.html)
- Date and author link of [announcement items](https://genomicsstandardsconsortium.github.io/gensc.github.io//news/2019/07/26/gsc22_bangkok_thailand.html) brought me to live GSC site

### [Past meetings](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/meetings.html)
- [GSC21, Vienna, Austria, May 20-23, 2019](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/meetings.html) returns 404 page not found.
- [GSC20, San Diego, USA, May 2-4, 2018](https://press3.mcs.anl.gov/gensc/meetings/gsc20/) brought me to live gsc site
- [GSC19, Brisbane, Australia, May 14-17, 2017](http://wiki.gensc.org/index.php?title=GSC_19) and many subsequent meetings throw errors:
```

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Strict Standards: Declaration of Skincavendish::initPage() should be compatible with Skin::initPage(OutputPage $out) in /var/sites/w/wiki.gensc.org/public_html/skins/cavendish.php on line 25

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902
```

### [About](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about.html)

- [Community](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/community.md) returns 404 page not found.

#### [Board](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/board-members.html)

1. All links below point to same page:

- Current Board Members
- Institutional Liasons to the GSC Board
- Advisory Board Members
- Former Board Members
- GSC Board Committees
- GSC Committees and Working Groups

2. `</tr> </tr> </tr> </tr> ` found under the `Current Board Members` at this [page](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/board-members.html#current-board)

#### [Publications](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/publication-list.html)

Meeting reports return errors:
- [GSC 16](http://wiki.gensc.org/index.php?title=GSC_16)

```
Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Strict Standards: Declaration of Skincavendish::initPage() should be compatible with Skin::initPage(OutputPage $out) in /var/sites/w/wiki.gensc.org/public_html/skins/cavendish.php on line 25

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match_all(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 876

Warning: Invalid argument supplied for foreach() in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 877

Warning: preg_replace(): Compilation failed: group name must start with a non-digit at offset 4 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 881

Warning: preg_match(): Compilation failed: group name must start with a non-digit at offset 8 in /var/sites/w/wiki.gensc.org/public_html/includes/MagicWord.php on line 902
```

#### [Funding](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/funding.html)

Links below return similar error as above:

- [Industry Annual Subscription Program](http://wiki.gensc.org/index.php?title=Industry_Sponsorship) 
- [Developer Group](http://wiki.gensc.org/index.php?title=Developer_Group)
- [Biodiversity Working Group](http://wiki.gensc.org/index.php?title=Biodiversity_Working_Group)

### [Project](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/projects.html) 

- [GSC Project Description Template](https://genomicsstandardsconsortium.github.io/pages/gsc-project-description-template.md) returns 404 site not found

#### [MIxS](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/standards/mixs.html)

- If you require further information about MIxS checklists, please visit the [compliance and implementation pages](http://gensc.org/mixs/mixs-compliance-and-implementation/), and [MIxS extensions pages](http://gensc.org/mixs/mixs-extensions/). <-- these two links brought me to GSC live page

### [Contact](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/contact.html)

- [GSC board page](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/board.md) returns 404 page not found
- [Compliance and interoperability](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/gensc-cig@googlegroups.com) returns 404 page not found

**Comment by only1chunts** [comment]
This is very helpful, thank you. I'm aware of the big issues with past meeting and the publications of the past meetings, this is a chunk of work that I might have to move out of this release as its not trivial! But all the other stuff you mention is great to have a list of, thank you so much for taking the time to do this.

**Comment by 79-6d** [comment]
You are more than welcome Chris!! Glad that I was able to help!! \o/

**Comment by only1chunts** [comment]
I've addressed most of the points (see **bold** below) There are number of the bigger issues that already have tickets open, so I will close this ticket now. Thanks again for checking things over.

> ### [Home](https://genomicsstandardsconsortium.github.io/gensc.github.io/index.html)
> * Date and author link of [announcement items](https://genomicsstandardsconsortium.github.io/gensc.github.io//news/2019/07/26/gsc22_bangkok_thailand.html) brought me to live GSC site
**Fixed - now takes you to the github version of the page**

> ### [Past meetings](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/meetings.html)
> * [GSC21, Vienna, Austria, May 20-23, 2019](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/meetings.html) returns 404 page not found.
**Fixed - this one has a page in new website so I've fixed the link to it**

> * [GSC20, San Diego, USA, May 2-4, 2018](https://press3.mcs.anl.gov/gensc/meetings/gsc20/) brought me to live gsc site
> * [GSC19, Brisbane, Australia, May 14-17, 2017](http://wiki.gensc.org/index.php?title=GSC_19) and many subsequent meetings throw errors:
**All other previous meetings need to have the pages created in GitHub site, so wont work until thats done**


> ### [About](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about.html)
> * [Community](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/community.md) returns 404 page not found.
**Fixed - community page was in the wrong location in repo! This page still needs work as its out of date and needs restructuring, created issue #50**

> #### [Board](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/board-members.html)
> 1. All links below point to same page:
> 
> * Current Board Members
> * Institutional Liasons to the GSC Board
> * Advisory Board Members
> * Former Board Members
> * GSC Board Committees
> * GSC Committees and Working Groups
**Fixed all links** 

> 1. `</tr> </tr> </tr> </tr> ` found under the `Current Board Members` at this [page](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/board-members.html#current-board)
**Fixed the table layout**

> ### [Project](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/projects.html)
> * [GSC Project Description Template](https://genomicsstandardsconsortium.github.io/pages/gsc-project-description-template.md) returns 404 site not found
**Fixed- link was wrong and the page was in the wrong place in repo**

> ### [Contact](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/contact.html)
> * [GSC board page](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/board.md) returns 404 page not found
**Fixed - page url was incorrect**
> * [Compliance and interoperability](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/gensc-cig@googlegroups.com) returns 404 page not found
**Fixed - it now uses mailto: command**


Still to be fixed:> 

> #### [Publications](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/publication-list.html)
> Meeting reports return errors:
**To be addressed in #26**


> #### [Funding](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/about/funding.html)
> Links below return similar error as above:
> 
> * [Industry Annual Subscription Program](http://wiki.gensc.org/index.php?title=Industry_Sponsorship)
> * [Developer Group](http://wiki.gensc.org/index.php?title=Developer_Group)
> * [Biodiversity Working Group](http://wiki.gensc.org/index.php?title=Biodiversity_Working_Group)
> 
**These will be addressed in #49**

#### [MIxS](https://genomicsstandardsconsortium.github.io/gensc.github.io/pages/standards/mixs.html)
> * If you require further information about MIxS checklists, please visit the [compliance and implementation pages](http://gensc.org/mixs/mixs-compliance-and-implementation/), and [MIxS extensions pages](http://gensc.org/mixs/mixs-extensions/). <-- these two links brought me to GSC live page
**This entire page still needs to be addressed #2 and #11**
